CMakeLists.txt

```diff
@@ -122,7 +122,7 @@ if (WITH_APPS)
 endif (WITH_APPS)
 
 if (WITH_PLUGINS)
-	FIND_PACKAGE(cJSON)
+	find_package(cJSON REQUIRED)
```

Comment:
It isn't required for all plugins, so I don't think this is correct.

Reply:
You are right, my bad.  
Actually the issue is that some app/client can require this dependency (`mosquitto_ctrl`, `sub_client_output`), thus if plugins are disabled it can lead to issues.  
I have moved the `find_package` out of the plugin check, but kept it optional.